pkg/util/fs_utils.go

```diff
@@ -53,6 +54,24 @@ func GetSize(path string) int64 {
 	return stat.Size()
 }
 
+//GetFileContents returns the contents of a file at the specified path
+func GetFileContents(path string) ([]string, error) {
+	stat, err := os.Stat(path)
+	if err != nil {
+		return nil, err
+	}
+
+	if stat.IsDir() {
+		return nil, errors.New("--filename is a directory, not a file")
+	}
+	contents, err := ioutil.ReadFile(path)
+
+	if err != nil {
+		return nil, err
+	}
+	return []string{string(contents)}, nil
```

Comment:
Why doesn't this return a normal string?

Reply:
the diff library I'm using wanted this format, but it probably makes more sense to move that to DiffFile